godel_robots/abb/godel_irb2400/godel_irb2400_support/config/scan_plan.yaml

```diff
@@ -0,0 +1,10 @@
+scan_plan:
+  scan_width: 0.01
+  traverse_spd: 0.0025
+  approach_distance: 0.15
+  overlap: 0.001
+  margin: 0
+  quality_metric: "rms"
```

Comment:
Well Gijs thought that denoting the active quality_metric via string, as seen here, was brittle. So I changed it to be an enumeration, but that means I'll need to switch this to a number and probably add a description so that readers know what they're looking at.